### PR TITLE
settings/translations: move locale to new folder, add extra folder an…

### PR DESCRIPTION
**Pull request by fuzzylogic2000**
…d path to use in forks

It's really basically like your PR, @phillimorland ! To use it in a fork, we then only uncomment from the base-settings and use make po-fork to mek the message files.

### CODE REVIEW

File: locale-source/locale/de_DE/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -23,8 +23,8 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
-#: adhocracy4/categories/templates/a4categories/includes/module_categories_form.html:3
-#: adhocracy4/categories/templates/a4categories/includes/module_categories_form.html:3
+#: /home/katharina/adhocracy-plus/node_modules/adhocracy4/adhocracy4/categories/templates/a4categories/includes/module_categories_form.html:3
+#: /home/katharina/adhocracy-plus/venv/lib/python3.8/site-packages/adhocracy4/categories/templates/a4categories/includes/module_categories_form.html:3
```
Comment (phillimorland) [inline locale-source/locale/de_DE/LC_MESSAGES/django.po:27]: does the 'gsed' check not work anymore?

Comment (fuzzylogic2000) [reply locale-source/locale/de_DE/LC_MESSAGES/django.po:27]: Oh, it should! Args. Why did I commit that? Thanks!

Comment (fuzzylogic2000) [reply locale-source/locale/de_DE/LC_MESSAGES/django.po:27]: @phillimorland

Comment (phillimorland) [reply locale-source/locale/de_DE/LC_MESSAGES/django.po:27]: it's easily done :p

Comment (fuzzylogic2000) [reply locale-source/locale/de_DE/LC_MESSAGES/django.po]: @phillimorland Yes, I think, I fucked up the rebase, but it's fixed now!

**Review by phillimorland — APPROVED** [review]
Super cool!

**Comment by phillimorland** [comment]
@fuzzylogic2000 that's a weird test to fail right?! tests/questions/test_question_templatetags.py

**Comment by fuzzylogic2000** [comment]
@phillimorland Yes, and the tests go through locally. I think we can merge. :crossed_fingers:

**Comment by phillimorland** [comment]
@fuzzylogic2000 ok living dangerously :p

**Pull request merged**